app/entities/paste-entity/paste.js

```diff
@@ -3,12 +3,12 @@ class Paste {
         this._id = data.id;
         this._name = data.name;
         this._text = data.text;
-        this._expiresAfter = data.expiresAfter;
+        this._expiresAfter = data.expires_after;
         this._visibility = visibility;
-        this._authorId = data.authorId;
-        this._createdAt = data.createdAt;
-        this._updatedAt = data.updatedAt;
-        this._deletedAt = data.deletedAt;
+        this._authorId = data.author_id;
+        this._createdAt = data.created_at;
+        this._updatedAt = data.updated_at;
+        this._deletedAt = data.deleted_at;
```

Comment:
Convert data **outside** of class constructor.
kebab_case is nonsense withtin camelCased style.
You've got kebab_case in DB? Okay, map it into camel **outside**. Somewhere far away from this place...